opcua/common/subscription.py

```diff
@@ -207,11 +202,11 @@ def _get_node(self, nodeid):
     def _get_filter_from_event_type(self, eventtype):
         eventtype = self._get_node(eventtype)
```

Comment:
Off-topic: This `_get_node` method is very useful and in a lot of cases we implement the same part of the code. Maybe we should think about creating one `common/tools.py` where we will put all this methods useful from different places.